scripts/xtensa-build-zephyr.sh

```diff
@@ -173,6 +173,10 @@ build_all()
 				PLAT_CONFIG='nxp_adsp_imx8'
 				RIMAGE_KEY='' # no key needed for imx8
 				;;
+			imx8x)
+				PLAT_CONFIG='nxp_adsp_imx8x'
+				RIMAGE_KEY='' # no key needed for imx8x
```

Comment:
```suggestion
				RIMAGE_KEY='ignored for imx8x'
```
The empty string confused me and cost me time while working on #4711

Reply:
Done